### PR TITLE
Updating references to docker.galasa.dev

### PR DESCRIPTION
**Pull request by Akyiaa**
Updating references to docker.galasa.dev
Story: https://github.com/orgs/galasa-dev/projects/3/views/1?filterQuery=akyiaa&pane=issue&itemId=41240946

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/src/main/java/dev/galasa/galasaecosystem/internal/properties/DockerRegistry.java [hunk 1]
```diff
@@ -21,12 +21,12 @@
...
  * @galasa.required No
  * 
- * @galasa.default docker.galasa.dev
+ * @galasa.default harbor.galasa.dev/docker_proxy_cache
  * 
  * @galasa.valid_values a valid hostname
  * 
  * @galasa.examples 
- * <code>galasaecosystem.docker.version=docker.galasa.dev</code>
+ * <code>galasaecosystem.docker.version=harbor.galasa.dev/docker_proxy_cache</code>
```
Comment (jadecarino) [inline galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/src/main/java/dev/galasa/galasaecosystem/internal/properties/DockerRegistry.java:29]: Hi Fiona, thanks for making these changes. Thinking about this - while harbor.galasa.dev/docker_proxy_cache is correct for us and the registry we use, a Galasa user might not use Harbor and instead might use their own registry or even Dockerhub. We should replace this with a more generic example to just demonstrate this needs to be a valid hostname of a Docker registry.
Therefore, I'd change the value to just be 'docker.io'.

Comment (jadecarino) [inline galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/src/main/java/dev/galasa/galasaecosystem/internal/properties/DockerRegistry.java:29]: This @galasa.examples line also contains the wrong key for the CPS property: it should say galasaecosystem.docker.registry. Please can you update?

File: galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/src/main/java/dev/galasa/galasaecosystem/internal/properties/DockerRegistry.java [hunk 2]
```diff
@@ -35,7 +35,7 @@ public static String get() throws GalasaEcosystemManagerException {
         try {
             String version = getStringNulled(GalasaEcosystemPropertiesSingleton.cps(), "docker", "registry") ;
             if (version == null) {
-                return "docker.galasa.dev";
+                return "harbor.galasa.dev/docker_proxy_cache";
```
Comment (jadecarino) [inline galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/src/main/java/dev/galasa/galasaecosystem/internal/properties/DockerRegistry.java:38]: docker.io

**Review by jadecarino — CHANGES_REQUESTED** [review]

File: galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/src/main/java/dev/galasa/galasaecosystem/internal/properties/DockerRegistry.java (hunk 1 above)
Comment (Akyiaa) [reply galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/src/main/java/dev/galasa/galasaecosystem/internal/properties/DockerRegistry.java:29]: That makes sense, will update, thanks

**Comment by galasa-team** [comment]
Build successful

**Review by jadecarino — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**